packages/edit-post/src/store/reducer.js

```diff
@@ -239,15 +239,39 @@ export function deviceType( state = 'Desktop', action ) {
 }
 
 /**
- * Reducer tracking whether the inserter is open.
+ * Reducer to set the block inserter panel open or closed.
  *
- * @param {boolean} state
- * @param {Object}  action
+ * Note: this reducer interacts with the list view panel reducer
+ * to make sure that only one of the two panels is open at the same time.
+ *
+ * @param {Object} state Current state.
+ * @param {Object} action Dispatched action.
  */
-function isInserterOpened( state = false, action ) {
+export function blockInserterPanel( state = false, action ) {
```

Comment:
@ellatrix has a PR that refactors this a bit which might conflict with the changes here.

Reply:
Thanks for the heads up!
The changes here are simple enough that I wouldn't expect the conflicts to be a big deal.
Just in case, do you happen to know which PR is that?

Reply:
#30301 I think